packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
```

Comment:
beautiful, i hope the radii weren't a pain to find

Reply:
I remeberd you talking about the weird choice of names ahah